web/src/containers/viewOnly/activities/Activity.js

```diff
@@ -242,7 +242,7 @@ const Activity = ({ activity, activityIndex }) => {
       </h3>
       <CostAllocateFFP
         aKey={activity.key}
-        activityIndex={0}
```

Comment:
Well, that'll do it, won't it.

Reply:
yep, it was a pretty easy fix, until I decided to add some other stuff in there with the testing. :D

Reply:
I was curious how long this bug has been around: 9 months!

Reply:
Yeah, we now have a test that will make sure that the tables are showing at least some of what they should